### PR TITLE
bottles and soda cans now break when hit by a bullet

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
add: hitting a bottle or a soda can with a bullet will make them break
/:cl:

now u can shoot cans n stuff in the courtyard

### CODE REVIEW

**Comment by Shapsy** [comment]
😭🥤🔫

**Comment by JMoldy** [comment]
Can cans take a few hits to break?

**Comment by Shadowflame909** [comment]
Can you add apples to this list.

Can we also make apples wearable

**Comment by ATH1909** [comment]
You missed that one, try another!

**Comment by OverlyGeneric** [comment]
Does this allow you to explode soda cans full of napalm and nitroglycerin? 🗿

**Comment by tralezab** [comment]
I'm actually furious someone beat me to the try another joke.

Anyway this is a great idea

**Comment by yoyobatty** [comment]
Add this for ammo too like mods for quake and ut99

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 1]
```diff
@@ -432,6 +438,16 @@
 		qdel(src)
 	..()
 
+/obj/item/reagent_containers/food/drinks/soda_cans/bullet_act(obj/item/projectile/P)
+	. = ..()
+	if(!(P.nodamage) && P.damage_type == BRUTE && !QDELETED(src))
+		var/obj/item/trash/can/crushed_can = new /obj/item/trash/can(src.loc)
+		crushed_can.icon_state = icon_state
+		qdel(src)
```
Comment (optimumtact) [inline code/modules/food_and_drinks/drinks/drinks.dm]: do this last please

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 2]
```diff
@@ -41,80 +38,61 @@
 		B.desc = "A carton with the bottom half burst open. Might give you a papercut."
 	B.name = "broken [name]"
 	transfer_fingerprints_to(B)
-
 	qdel(src)
 	target.Bumped(B)
 
 /obj/item/reagent_containers/food/drinks/bottle/attack(mob/living/target, mob/living/user)
-
```
Comment (optimumtact) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: please undo your changes to this file

File: code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm [hunk 3]
```diff
@@ -91,7 +91,7 @@
 /obj/item/reagent_containers/food/drinks/drinkingglass/filled/nuka_cola
 	name = "Nuka Cola"
 	list_reagents = list("nuka_cola" = 50)
-	
+
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm]: this file too

Comment (Tlaltecuhtli) [reply code/modules/food_and_drinks/drinks/drinks/drinkingglass.dm]: huh?

**Pull request merged**